config/samples/osp-director_v1beta1_openstackdeploy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackDeploy
+metadata:
+  name: openstackdeploy-sample
+spec:
+  configVersion: bar
+  configGenerator: foo
```

Comment:
should list optional imageURL param?

Reply:
well, I suppose I'd like to have the example just rely on our default. I'd actually like to go the other way and have most examples remove the imageURLs

Reply:
ok, maybe we should have more samples. Non defaults in a subdir, like for kuttl tests where we use/validate some more parameters not set per default